test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,24 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+
+        // only one object
+        assertTrue(Utils.isAnyNull(((Object) null)));
+        assertFalse(Utils.isAnyNull(""));
+        assertFalse(Utils.isAnyNull(1));
+        assertFalse(Utils.isAnyNull("abc"));
+
+        // multiple objects
+        assertFalse(Utils.isAnyNull("abc", "abc"));
+        assertFalse(Utils.isAnyNull("abc", "", "abc", "ABC"));
+        assertFalse(Utils.isAnyNull("", "abc", "a", "abc"));
+        assertFalse(Utils.isAnyNull(1, new Integer(1)));
+        assertTrue(Utils.isAnyNull(null, 1, new Integer(1)));
+        assertTrue(Utils.isAnyNull(null, null));
+        assertTrue(Utils.isAnyNull(null, "a", "b", null));
```

Comment:
instead of grouping test cases as per the number of arguments, it is probably better to group as per functionality. Also, the most ideal case is to separate them out and make them specific cases with 3-part naming scheme mentioned in the tutorial.